src/plugins/discover/public/application/doc_views_links/doc_views_links_types.ts

```diff
@@ -10,7 +10,7 @@ import { IndexPattern } from '../../../../data/public';
 export interface DocViewLink extends EuiListGroupItemProps {
   href?: string;
   order: number;
-  generateurlfn?(renderProps: any): string;
+  generateurlcb?(renderProps: any): string;
```

Comment:
Any reason this property isn't camel cased?

Reply:
Not really, changed.